src/app_service/service/settings/service.nim

```diff
@@ -30,6 +32,14 @@ type
     statusType*: StatusType
     text*: string
 
+type
+  SettingProfilePictureShowToArgs* = ref object of Args
```

Comment:
@PascalPrecht you could name this like `SettingProfilePictureArgs* = ref object of Args` and share the same type among multiple signals, no need to have new type every times, as signals will be handled by their names (names should be unique).

So please update this and remove unnecessary type below.

Reply:
Thanks @saledjenic , will do!

Reply:
This is now updated!